src/core/default/commands/relationaldb/shell.py

```diff
@@ -143,12 +146,22 @@ def __init__(
         self, fmt: fmt, cluster_arn: str, secret_arn: str, database_name: str
     ) -> None:
         super().__init__()
+        history_location = os.path.join(Workspace.instance().settings.INTERMEDIATE_FOLDER_LOCATION, 'dbshell')
```

Comment:
small change, but could you move this to lines 55 then pass in the history location as a parameter to the `interactive shell` class